adam-apis/src/test/scala/org/bdgenomics/adam/apis/java/JavaADAMContextSuite.scala

```diff
@@ -26,19 +26,73 @@ import org.bdgenomics.formats.avro.AlignmentRecord
 
 class JavaADAMContextSuite extends ADAMFunSuite {
 
-  sparkTest("can read a small .SAM file") {
+  sparkTest("can read and write a small .SAM file") {
     val path = copyResource("small.sam")
-    val ctx = new JavaADAMContext(sc)
-    val reads = ctx.loadAlignments(path)
-    assert(reads.jrdd.count() === 20)
+    val aRdd = sc.loadAlignments(path)
+    assert(aRdd.jrdd.count() === 20)
+
+    val newRdd = JavaADAMReadConduit.conduit(aRdd, sc)
```

Comment:
Ah right, I missed the language jump!

It might be useful to have a Java version of `ADAMFunSuite` for downstream users of the API module, and then the unit tests for `JavaADAMContext` could bridge both languages.  This would require running both junit and scalatest test runners (which shouldn't be a problem) and would allow us to create a `test-jar` artifact for use downstream.

For now I would suggest making the `*Conduit` classes package private, and we can expand unit test support in Java as part of #855.